static/src/components/builder/components/MultiselectInput/MultiselectForm.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+  <div class="row">
+    <h4>Multiselect Settings</h4>
+    <div class="form-group">
+      <input
+        id="add-label"
+        v-model="labelAdded"
+        type="checkbox"
+      >
+      <label for="add-label">
+        Add Multiselect Label
+      </label>
+      <input
+        v-if="labelAdded === true"
+        v-model="label"
+        class="form-control form-control-sm"
+        type="text"
+      >
+    </div>
+    <label
+      class="block-label"
+    >
+      Answer field name | <span class="label-tip">The field where the worker's answer is stored. Can be JSON path like a.b.c.</span>
+      <input
+        id="pyb-answer"
+        v-model="pybAnswer"
+        class="form-control form-control-sm"
+        type="text"
+      >
+    </label>
+    <validator
+      :validations="validations"
+      :validation-options="filteredValidations"
+      :update-validations="updateValidations"
+    />
+    <div class="form-group">
+      <label
+        class="col-labels"
+        for="initial-value"
+      >
+        Initial Value | <span class="label-tip">Must be one of the configured multiselect values.</span>
```

Comment:
Is the initial value field a single-select dropdown or a multi-select dropdown?

Reply:
I don't really understand the question. I will follow up via IB!